tabpy-server/tabpy_server/handlers/evaluation_plane_handler.py

```diff
@@ -112,5 +109,4 @@ def call_subprocess(self, function_to_evaluate, arguments):
         else:
             future = self.executor.submit(_user_script, restricted_tabpy,
                                           **arguments)
-        ret = yield future
-        raise gen.Return(ret)
+        return asyncio.wrap_future(future)
```

Comment:
Why not ?
async def call_subprocess(self, function_to_evaluate, arguments):
...
return await future